File: 2025-03-01/python/08.py
#
# For a given number, return the sum of the 3rd, 4th digit without using extra variables
# Preferred to do in C


def solve(num: int):
    return int(str(num)[2]) + int(str(num)[3])


num = 5321641
print(num)
print(solve(num))

File: 2025-03-01/c/08.c
#include <stdio.h>
// For a given number, return the sum of the 3rd, 4th digit without using extra variables
void main(){
    int n = 5321641;
    while(n > 9999){
        n = n / 10;
    }
    printf("%d\n", n % 10 + (n / 10) % 10);
}